PROJECT

```diff
@@ -1,8 +1,14 @@
 domain: windowsmachineconfig.openshift.io
-layout: go.kubebuilder.io/v3
+layout:
+- go.kubebuilder.io/v3
```

Comment:
Why does `go.kubebuilder.io/v3` need to move below `layout:`?

Reply:
This is just a generated file change, I'm not sure why it wasn't this way in the first place.